src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend_utils.py

```diff
@@ -24,13 +24,14 @@ def _get_device(options) -> Optional[Any]:
             + device
             + " is not in the list of OpenVINO Available Devices"
         )
-
+    else:
+        device = "CPU"
```

Comment:
why not AUTO?

Reply:
It makes sense but we haven't validated with AUTO for all the models so i'm not sure if this will affect anything. We can definitely test this out and include it in the next release.